spiffworkflow-backend/src/spiffworkflow_backend/models/message_instance.py

```diff
@@ -45,21 +46,22 @@ class MessageInstanceModel(SpiffworkflowBaseDBModel):
     __tablename__ = "message_instance"
 
     id: int = db.Column(db.Integer, primary_key=True)
-    process_instance_id: int = db.Column(ForeignKey(ProcessInstanceModel.id), nullable=False)  # type: ignore
-    message_model_id: int = db.Column(ForeignKey(MessageModel.id), nullable=False)
-    message_model = relationship("MessageModel")
-    message_correlations_message_instances = relationship(
-        "MessageCorrelationMessageInstanceModel", cascade="delete"
-    )
-
+    process_instance_id: int = db.Column(ForeignKey(ProcessInstanceModel.id), nullable=True)  # type: ignore
```

Comment:
When will we have a message instance and not a process instance?

Reply:
When as message is sent in over the API.

Reply:
But then I would think you create the process instance and then the message instance. I think it should always need a process instance.

Reply:
I changed it so that we are now recording the API Send Message in the table, not just the receive message - so that API Send Message record in the table was not sent by a process instance.  I think this is useful information to have.  That a message was sent from outside the system.